### PR TITLE
LimeView.qml Clean Up

### PR DESCRIPTION
**Pull request by bawigga**
No new functionality here, just general cleanup and refactoring.

I jumped into the LimeView.qml file to help contribute this morning, but quickly found it hard to follow what was going on. This was mostly due to single character variable names and unclear javascript functions. This PR cleans up a lot of that code and hopefully makes it more approachable for future contributors.

### CODE REVIEW

**Comment by erbridge** [comment]
LGTM. @zoli want to confirm?

**Comment by zoli** [comment]
LGTM, Except for the comments.

**Comment by bawigga** [comment]
@zoli Ok, thanks for the heads up on the comments, I'll get those fixed.

At first I thought there was a degradation in cursor positioning also. I went back and double checked how it worked before my changes and it felt the same. There's an issue on short lines of text with a tab at the beginning, where clicking at the start of the text puts the cursor at the end of the line. Got any more details on what you're seeing?

**Comment by zoli** [comment]
If the positioning problem is just for me we can merge this there is no problem, @erbridge can you test this?

**Comment by erbridge** [comment]
Cursor positioning is right for me on master too, but I'm also on Arch Linux. I wonder if it's and OS X vs Linux issue...

**Comment by erbridge** [comment]
Let's keep the cursor fix to a separate issue (#502).

**Comment by bawigga** [comment]
Thanks guys! I think all the feedback has been implemented.

**Comment by zoli** [comment]
LGTM, Thanks.
